src/Components/Radio/Radio.component.jsx

```diff
@@ -164,14 +164,15 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const statusBank = {
     EMPTY: 'Please Insert a Cassette!',
     PLAYING: `Now Playing: ${nowPlaying}`,
-    LOADED: `Press Play?`
+    LOADED: 'Press Play?',
+    LOADING: 'Loading...1 sec'
```

Comment:
The transition from LOADING to LOADED breaks the audio stream on both tapes for me